composer.json

```diff
@@ -67,12 +68,10 @@
             "./phpstan.phar analyse src tests --level=2 -c phpstan.neon --no-interaction --no-progress"
         ],
         "cs:check": [
-            "wget -nc http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar",
```

Comment:
Sorry but I don't like to depend on static analyzer tools. PHAR distribution is what is really needed here. There are many reason, at least:

- this is not a dependency of Infection, just tool that analyzes CS
- as a package, it can conflict with other dependencies
- as a package, it is locked to specific version